conformance/Makefile

```diff
@@ -48,7 +48,6 @@ load-images: ## Load NGF and NGINX images on configured kind cluster
 .PHONY: prepare-ngf-dependencies
 prepare-ngf-dependencies: update-ngf-manifest ## Install NGF dependencies on configured kind cluster
 	./scripts/install-gateway.sh $(GW_API_VERSION)
-	kubectl wait --for=condition=available --timeout=60s deployment gateway-api-admission-server -n gateway-system
```

Comment:
It was a quick fix for testing purposes - they removed default installation of Gateway API admission server and related resources (because of the move to using CEL validation instead). We'll actually want to update to optionally install these resources for cases where the k8s version is <1.26. See https://github.com/kubernetes-sigs/gateway-api/issues/2400